src/Cms/Page.php

```diff
@@ -1467,7 +1467,8 @@ public function toArray(): array
      */
     protected function token(): string
     {
-        return sha1($this->id() . $this->template());
+        $salt = $this->kirby()->option('content.salt', $this->root());
+        return sha1($this->id() . $this->template() . $salt);
```

Comment:
This should use an HMAC, otherwise the salt could potentially be circumvented.

```suggestion
        return hash_hmac('sha1', $this->id() . $this->template(), $salt);
```